### PR TITLE
Net benefit

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#264 #266 
#### What does this implement/fix? Explain your changes.
- Handler functions moved to `pycm_handler.py`
- Error objects moved to `pycm_error.py`
- Verified tests references updated
- Verified tests moved to `verified_test.py`
- Test system modified 
- `CONTRIBUTING.md` updated
- Namespace optimized
- `Conda Managing Package` broken link fixed
- `.gitattributes` updated

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@sadrasabouri @alirezazolanvari 

Hi folks ;-)
I made some important edit in PyCM source code so please check carefully and leave some comment ;-)
@alirezazolanvari please reformat added references (`64`,`65`,`66`) in `README.md`  - `Net-Benefit` branch and then inform me ;-)

**Notice 1** : Do not hurry, we have enough time ;-)
**Notice 2** : This PR has 2 reviewer, so please don't merge this PR to  `dev` branch before finishing the discussion
**Notice 3** : @sadrasabouri I invited you as a new collaborator, please accept. After that I will assign you as reviewer 

Best Regards
Sepand Haghighi

**Comment by sepandhaghighi** [comment]
**Update :** 

I made some new minor edits : 
1. `Conda Managing Package` broken link fixed #266 
2. `.gitattributes` updated
3. `verified_test.py` mentioned in `CONTRIBUTING.md`

**Comment by sepandhaghighi** [comment]
@sadrasabouri @alirezazolanvari 
Just a friendly reminder! ⏲

File: pycm/pycm_class_func.py [hunk 1]
```diff
@@ -5,6 +5,27 @@
 from .pycm_interpret import *
 
 
+def NB_calc(TP, FP, POP, w):
```
Comment (sadrasabouri) [inline pycm/pycm_class_func.py]: Shouldn't we set a default value like 1 for w?
`w=1`

Comment (sepandhaghighi) [reply pycm/pycm_class_func.py]: Good point 💯 
I added a default value for `w` in `NB` method https://github.com/sepandhaghighi/pycm/pull/265/commits/14dd45a2c7b93e96d516e849885c592d69678316

**Comment by sepandhaghighi** [comment]
@alirezazolanvari 
Just another friendly reminder! ⏲

**Comment by sepandhaghighi** [comment]
@alirezazolanvari @sadrasabouri Thanks ;-)

**Pull request merged**